tracex_project/tracex/logic/utils.py

```diff
@@ -1,26 +1,36 @@
 """Module providing various utility functions for the project."""
 import os
-from io import StringIO
-from pathlib import Path
```

Comment:
Where these just unused imports? Since you did not delete or change any code, I am a bit confused why you are deleting imports.

Reply:
I was reordering it due to some import ordering pylint fails